src/cmd/manage_nsfs.js

```diff
@@ -492,11 +492,14 @@ async function fetch_existing_account_data(target) {
         source = await get_config_data(account_path, true);
     } catch (err) {
         dbg.log1('NSFS Manage command: Could not find account', target, err);
-        if (is_undefined(target.name)) {
-            throw_cli_error(ManageCLIError.NoSuchAccountAccessKey, target.access_keys[0].access_key);
-        } else {
-            throw_cli_error(ManageCLIError.NoSuchAccountName, target.name);
+        if (err.code === 'ENOENT') {
+            if (is_undefined(target.name)) {
+                throw_cli_error(ManageCLIError.NoSuchAccountAccessKey, target.access_keys[0].access_key);
+            } else {
+                throw_cli_error(ManageCLIError.NoSuchAccountName, target.name);
+            }
         }
+        throw err;
```

Comment:
@romayalon so we will catch the rest of the error in main block, right?

Reply:
yes, exactly.
without it, even if there was an error we will say that the operation succeeded.